src/System.Runtime.Numerics/tests/BigInteger/cast_from.cs

```diff
@@ -479,7 +479,7 @@ public static void RunSingleExplicitCastFromBigIntegerTests()
             VerifySingleExplicitCastFromBigInteger(Single.MaxValue, bigInteger);
 
             // Single Explicit Cast from BigInteger: Random value > Single.MaxValue
-            bigInteger = GenerateRandomBigIntegerGreaterThan(Single.MaxValue * 2, s_random);
+            bigInteger = GenerateRandomBigIntegerGreaterThan((double)Single.MaxValue * 2, s_random);
```

Comment:
@weshaggard it was a bug in the native compiler.  Roslyn properly adheres to the spec behavior here.  

CC @gafter